src/client/components/ChatBox.js

```diff
@@ -52,28 +53,50 @@ export default class ChatBox extends Component {
 
   sendMessage () {
     console.log('[ChatBox] sendMessage to room: ' + this.props.lecture.room)
-    // Setting msg.text to written input
-    var msg = {
-      text: this.state.text,
-      lecture: {
-        id:  JSON.stringify(this.props.lecture.id),
-        code: this.props.lecture.course.code,
-        room: this.props.lecture.room,
-      }
-    }
+    if (this.validateMessage(this.state.text)) {
+      // Setting msg.text to written input
+      var msg = {
+        text: this.state.text,
+        lecture: {
+          id:  JSON.stringify(this.props.lecture.id),
+          code: this.props.lecture.course.code,
+          room: this.props.lecture.room,
+        },
+        errorText: ''
 
-    // Emtpy input field
-    this.setState({text: ''})
+      }
+       // Emtpy input field
+      this.setState({text: ''})
 
-    socket.emit('new-message', msg)
-    console.log('[ChatBox] After sending message')
+      socket.emit('new-message', msg)
+      console.log('[ChatBox] After sending message')
+   
+    }
+    
   }
   // Listen and update field dynamically when something is written
   changeHandler (e) {
     this.setState({ text: e.target.value })
     console.log('[ChatBox] changeHandler')
   }
 
+  validateMessage (msg) {
```

Comment:
The error message overflows when the message box has 2 lines or more of text:
![image](https://cloud.githubusercontent.com/assets/23553119/25062777/9e8efee4-21cb-11e7-89ae-29b06b17450f.png)